portal/client/src/app/auth/signup/signup.component.ts

```diff
@@ -29,16 +30,20 @@ export class SignupComponent implements OnInit {
       return
     }
     if (event.form.value.password != event.form.value.passwordVerif) {
-        this.message = "your password is not the same than your password confirmation"
+        this.messageError = "your password is not the same than your password confirmation"
         return
     }
     this.httpService.signup(event.form.value.username, event.form.value.password, event.form.value.email).subscribe(
       data => {
-        this.message = "Your account is created, you are going to receive an email. Confirmed you received it, before sign in"
+        this.message = "Your account is created, you are going to receive an email to validate your account"
```

Comment:
@freignat91 We do set the environment variable [here](https://github.com/appcelerator/amp/blob/master/cli/command/cluster/create.go#L52) and that's how amplifier also gets it. I don't want this to hold up the PR, so I'll open an issue so that we make sure to the feedback is correct for the current configuration.